ai-prompts/mcp.txt

```diff
@@ -18,6 +18,7 @@ Response guidelines:
 - Invoke provided tools only when necessary
 - Respond short and concisely
 - Answer only questions that are related to OpenOps, FinOps, Cloud Providers, Cloud Infrastructure tools, and Cloud Ops.
+- if there is a link provided to the docs, use the correct URL in the response, it should start with https://docs.openops.com/
```

Comment:
what happens if you ask it to give a link to aws docs?

Reply:
worked fine, but I'll make it more specific